src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1388,6 +1388,54 @@ def test_aks_nodepool_abort(self, resource_group, resource_group_location):
             self.check('powerState.code', 'Running')
         ])
 
+    @live_only()
```

Comment:
May I ask why do you mark these tests as `@live_only()`?

Reply:
Probably could remove this decorator (`@live_only()`), as you've already committed the recording files.

Reply:
Removed